fixtures/ios_app_with_appclip/App/Configs/Info.plist

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Acceptance test fixtures have a maintenance cost that I'd avoid as much as possible. In this case we can use `.default` info plists and remove this file.